pol-core/bscript/compiler/astbuilder/ExpressionBuilder.cpp

```diff
@@ -274,6 +290,41 @@ std::vector<std::unique_ptr<Expression>> ExpressionBuilder::expressions(
   return expressions;
 }
 
+std::vector<std::unique_ptr<Expression>> ExpressionBuilder::expressions(
+    std::vector<EscriptGrammar::EscriptParser::InterpolatedStringPartContext*> ctx )
+{
+  std::vector<std::unique_ptr<Expression>> expressions;
+  // FIXME: optimize values
+  // FIXME: double-curleys
+  for ( auto interstringPart_ctx : ctx )
+  {
+    if ( auto expression_ctx = interstringPart_ctx->expression() )
+    {
+      std::unique_ptr<Expression> expr = expression( expression_ctx );
+      if ( auto format = interstringPart_ctx->FORMAT_STRING() )
+      {
+        expr = formatted_string( std::move( expr ), format );
```

Comment:
If there's a format string, create a new `FormattedString` ast node, using the interstring expression as its child.